wallet/wallet-cli-commands/src/lib.rs

```diff
@@ -680,6 +681,26 @@ pub enum WalletCommand {
     #[clap(hide = true)]
     GenerateBlocks { block_count: u32 },
 
+    /// For each block height in the specified range, find timestamps where staking is/was possible
+    /// for the given pool.
+    ///
+    /// `min_height` must not be zero; `max_height` must not exceed the best block height plus one.
+    ///
+    /// If `check_all_timestamps_between_blocks` is "no", `seconds_to_check_for_height + 1` is the number
+    /// of seconds that will be checked at each height in the range.
+    /// If `check_all_timestamps_between_blocks` is "yes", `seconds_to_check_for_height` only applies to the
+    /// last height in the range; for all other heights the maximum timestamp is the timestamp
+    /// of the next block.
+    #[clap(name = "node-find-timestamps-for-staking")]
+    #[clap(hide = true)]
+    FindTimestampsForStaking {
+        pool_id: String,
+        min_height: BlockHeight,
+        max_height: BlockHeight,
+        seconds_to_check_for_height: u64,
+        check_all_timestamps_between_blocks: YesNo,
+    },
```

Comment:
2discuss: I'm also not sure about the cli interface; perhaps, `pool_id` shouldn't be here and the search should always be performed for all pools in all accounts? (or may be we need an additional command for that).